### ISSUE TITLE
app-misc/electricsheep-9999-r8 can't find boost/filesystems/convenience.hpp

### ISSUE THREAD

**Issue by ExecutorElassus** [stefantalpalaru/gentoo-overlay#146]
Showing just the error:

```
In file included from Player.h:12,
                 from Voting.cpp:17:
lua_playlist.h:20:17: fatal error: boost/filesystem/convenience.hpp: No such file or directory
   20 | #include        "boost/filesystem/convenience.hpp"
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:667: Voting.o] Error 1
make[1]: *** Waiting for unfinished jobs....
In file included from Player.h:12,
                 from Player.cpp:36:
lua_playlist.h:20:17: fatal error: boost/filesystem/convenience.hpp: No such file or directory
   20 | #include        "boost/filesystem/convenience.hpp"
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:667: Player.o] Error 1
In file included from Player.h:12,
                 from client.h:10,
                 from main.cpp:16:
lua_playlist.h:20:17: fatal error: boost/filesystem/convenience.hpp: No such file or directory
   20 | #include        "boost/filesystem/convenience.hpp"
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:667: main.o] Error 1
make[1]: Leaving directory '/var/tmp/portage/app-misc/electricsheep-9999-r8/work/electricsheep-9999/client_generic/Client'
make: *** [Makefile:516: all-recursive] Error 1
```
In my build directory, I only see 
`/var/tmp/portage/app-misc/electricsheep-9999-r8/work/electricsheep-9999/client_generic/boost/osx/`, which is empty.

**Comment by stefantalpalaru** [comment]
What's the output of `eix -se boost`?

**Comment by ExecutorElassus** [comment]
```
# eix -se boost
[?] dev-libs/boost
     Available versions:  1.52.0-r7(0/1.52) (~)1.53.0-r1(0/1.53) (~)1.54.0-r1(0/1.54) 1.55.0-r2(0/1.55.0)^t 1.56.0-r1(0/1.56.0)^t (~)1.57.0(0/1.57.0)^t (~)1.58.0-r1(0/1.58.0)^t {abi_ppc_32 abi_ppc_64 context debug doc icu mpi +nls python static-libs +threads tools ABI_MIPS="n32 n64 o32" ABI_S390="32 64" ABI_X86="32 64 x32" PYTHON_TARGETS="python2_7 python3_3 python3_4"}
     Installed versions:  1.85.0-r1(0/1.85.0.1)^t(03:42:25 PM 06/12/2024)(bzip2 context nls python stacktrace zlib -debug -doc -icu -lzma -mpi -numpy -tools -zstd ABI_MIPS="-n32 -n64 -o32" ABI_S390="-32 -64" ABI_X86="32 64 -x32" PYTHON_TARGETS="python3_10 python3_11 python3_12")
     Homepage:            http://www.boost.org/
     Description:         Boost Libraries for C++
```

**Comment by stefantalpalaru** [comment]
Fixed.
